test/jdk/java/lang/RuntimeTests/loadLibrary/LoadLibraryTestAIX.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024, IBM Corporation. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/**
+ * @test
+ * @bug 8319516
+ * @requires os.family == "aix"
+ * @run  main/native/othervm -Djava.library.path=/usr/lib LoadLibraryTestAIX
+ */
+
+public class LoadLibraryTestAIX {
+    public static void main(String[] args) throws Exception {
+        String libraryName="perfstat";
+        try {
+            System.loadLibrary(libraryName);
+        } catch (Exception e) {
+            throw new RuntimeException("LoadLibraryTestAIX : could not load libperfstat.a"+e);
+        }
```

Comment:
Nit: the catch clause is not strictly needed.   `UnsatisfiedLinkError` is an unchecked exception.   The test should fail if you run it with JDK without this fix.

Reply:
The test should verify if `/usr/lib/libperstat.a` is present and also `/usr/lib/libperfstat.so` is not present.

Do you expect all AIX machines do not have `/usr/lib/libperfstat.so`?

Reply:
Yes it is expected.  the hotspot code has harcoded it to .a file.

Reply:
In this case, the test can verify if `/usr/lib/libperstat.a` is present and `/usr/lib/libperfstat.so` is not present; otherwise, the test should throw `RuntimeException`.   This also serves a good documentation purpose showing what this test expects.